### ISSUE TITLE
What can be included in excludes.

### ISSUE THREAD

**Issue by jindalpratik** [garethgeorge/backrest#104]
I have looked at the backrest wiki and hovered over the excludes option but there is no indication what I can put in the option to exclude file and folders or if I can use regex expressions or not. I am pretty used to using regex in duplicati and kopia(which even supports a .ignore file to excude files and fodlers) but don't know if I can use those in backrest or if I have to use complete paths of each folder and file I would like to exclude.

**Comment by garethgeorge** [comment]
Can definitely add some text to the docs on this, to fill in for the meantime. Excludes are directly mapped to `--excludes` flag in restic. The restic docs talk a bit more about what patterns work: https://restic.readthedocs.io/en/latest/040_backup.html#excluding-files

**Comment by jindalpratik** [comment]
> Can definitely add some text to the docs on this, to fill in for the meantime. Excludes are directly mapped to `--excludes` flag in restic. The restic docs talk a bit more about what patterns work: https://restic.readthedocs.io/en/latest/040_backup.html#excluding-files 

Thanks for the info this helps a lot, a link to the docs on hover will probably be nice too.

**Comment by mvuorre** [comment]
I can't see an `--excludes` flag in the restic docs. Do you mean `--exclude` (and so only filenames/patterns are allowed)? Or can I also use e.g. `--exclude-larger-than size`?

**Comment by garethgeorge** [comment]
tl;dr is at the moment "--exclude-larger-than size" isn't supported. 

The longer answer is that I'm aiming to support more generic restic options e.g. `--exclude-larger-than` with repo level flags. At the moment repo-level flags only support presence flags (e.g. flags without values). A little bit of feature work is needed but it should be possible for me to support key value flags e.g. `--exclude-larger-than size`.

**Comment by garethgeorge** [comment]
Repo level flags are now supported and parsed correctly, they should address this use case!
